### DISCUSSION TITLE
Raspberry Pico RP2040 support works well

### DISCUSSION THREAD

**Discussion by sail81** [Arduino-IRremote/Arduino-IRremote#953] [Show and tell]
Hi folks,

thank you for your great work on this library. It is much appreciated. Recently I checked out the minimal NEC receiver with a Raspberry Pico RP2040 and a Vishay TSOP38238 IR receiver. After changing the one pin the example just worked out of the box. In my application I am using it to control Kodi running on an RPI4B with my NEC TV remote control. The Raspberry Pico I configured as a USB HID keyboard device applying the TinyUSB stack. The experience was quite a positive surprise. It is like pressing hard wired buttons. The Flirc USB dongle I was using previously is way slower.

Many thanks again!

**Comment by toseef-ahmd** [comment]
@sail81 I'm trying to configure RP2040 with the latest version of the library. But it is not working for me. Can you share how did you set it up or what changes did you make?
